languages/python/GAMS/how_to_guide/gams_python_api_how_to_guide.md

```diff
@@ -0,0 +1,81 @@
+# Create Anaconda Environment
+
+First, create an Anaconda environment that you will use for all your experiments related to gams in python. Follow the steps below.
+
+### 1. Log in to Eagle
+
+```
+ssh <username>@eagle.hpc.nrel.gov
+```
+where, 'username' is your NREL HPC (Eagle) user name.
+
+### 2. Set up Anaconda environment
+
+The repo provides the gams_python_api.yml file. Use it to create a new Anaconda environment at a directory of your choose. 
+
+There are three main directories on Eagle where you can install the new environment, namely /home, /scratch, and /projects. Please go to NREL HPC resources page to find more information about the various Eagle directories and how to create new Anaconda environments.
+
+Example:
+
+Start by creating a subdirectory /scratch/$USER/github-repos/, cd there and clone this repo. 
+
+Assuming you want to install your new environment in your scratch directory, you may want to create a directory that will contain all your Anaconda environments, e.g. /scratch/$USER/conda-envs/:
+
+```
+conda env create --prefix=/scratch/$USER/conda-envs/gams_python -f gams_python_api.yml
+```
+The installation may take a couple of minutes, wait until it is complete and you get a done message.
+
+# Load GAMS and Set up Environment Parameters
+
+Now, activate your Anaconda environment, load the appropriate GAMS version available on Eagle and establish the required environment varables. Follow the steps below.
```

Comment:
I suggest including a bit more explanation. Something along the lines of "This Conda environment includes a basic Python environment. The GAMS API modules are then loaded directly from the module loaded on Eagle" 

Or somehow convey that the module load is actually where the API files are coming from and not a pip/conda install in the conda yaml file.

Reply:
Added this as a note in the same place.